src/core/lib/security/credentials/tls/grpc_tls_certificate_distributor.h

```diff
@@ -40,8 +40,7 @@ struct grpc_tls_identity_pairs {
 };
 
 // TLS certificate distributor.
-struct grpc_tls_certificate_distributor
-    : public grpc_core::RefCounted<grpc_tls_certificate_distributor> {
+class TlsCertificateDistributor {
```

Comment:
Should we consider changing the filename too?

Reply:
I don't think we need to change the filenames - they have their own convention, for example the `CrlProviders` are called `grpc_crl_provider.h`, then the internal tls specific impl is `grpc_tls_crl_provider.{h/cc}`.